src/components/resource-section/add-evaluation/AddEvaluation.js

```diff
@@ -9,17 +9,19 @@ import styles from './AddEvaluation.module.scss';
 const AddEvaluation = ({ formOptions }) => {
   const [evaluation, setEvaluation] = useState(formOptions.evaluation);
   const [comment, setComment] = useState(formOptions.comment);
-  const [evaluated, setEvaluated] = useState(formOptions.evaluated);
+
+  useEffect(() => {
```

Comment:
This code smells weird. It works regarding updating the rating of the active resource. However, it seems to be another refactor re-designing the use of mutate and related data to update more directly.